tikv-client/src/main/java/com/pingcap/tikv/codec/TableCodec.java

```diff
@@ -125,22 +133,76 @@ public static Handle decodeHandle(byte[] value, boolean isCommonHandle) {
   // 		|     Length:     1         | 2           | len(CHandle)
   // 		|
```

Comment:
The options comment also need to update.

Reply:
@qidi1 Thanks for your review. I will update comment here.